### ISSUE TITLE
"Resguardo contra las sombras" bug

### ISSUE THREAD

**Issue by Lahu** [RoDNet/Bugtracker-Infinium-4.0.6a#472]
La skill Resguardo contra las sombras del brujo desaparece de la barra de skill y deja de funcionar al poco rato de entrar al servidor

**Comment by jandrox** [comment]
Correcto, yo he puesto un tiket creyendo que era mi pj Vidarius que estaba bugeado de algún modo pero veo que es en general, y esto deja el talento de Resguardo Abisal inútil dejándonos a los brujos con la defensa por los suelos :(
